exec/cnex/module.c

```diff
@@ -114,6 +123,15 @@ TModule *module_loadModule(TExecutor *exec, const char *name, unsigned int modul
     path_readModule(r);
     bytecode_loadBytecode(r->bytecode, r->code, r->codelen);
 
+    // Allocate the global function pointer cache for this module
```

Comment:
This block of code looks substantially similar to the above block for the main module. Is there a way to consolidate this code into one function?

Reply:
Yea.  Removed it from module entirely, and moved it to cnex.c.